tests/test_dialects.py

```diff
@@ -1749,6 +1749,11 @@ def test_oracle(self):
             "SELECT x FROM y LIMIT 10",
             read="oracle",
         )
+        self.validate(
```

Comment:
There's probably a more generic way to test this, since its not necessarily specific to Oracle